lib/step_definitions/given.js

```diff
@@ -0,0 +1,66 @@
+const {
+    Given,
+} = require("cypress-cucumber-preprocessor/steps");
+const env = require('../environment_variables.js')
+
+module.exports.register = function () {
+    Given(/^the user is logged into the CES$/, function () {
+        cy.fixture("testuser_data").then(function (testUser) {
+            cy.login(testUser.username, testUser.password)
+        })
+    });
+
+    Given(/^the user is logged out of the CES$/, function () {
+        cy.logout()
+    });
+
+    Given(/^the user is not member of the admin user group$/, function () {
+        // default behaviour
+    });
+
+    Given(/^the user is member of the admin user group$/, function () {
+        cy.fixture("testuser_data").then(function (testUser) {
+            cy.promoteAccountToAdmin(testUser.username)
+        })
+    });
+
+    Given(/^the user has no internal dogu account$/, function () {
+        // default behaviour
+    });
```

Comment:
Maybe we could ensure that the default behaviour works as expected by executing the necessary steps.